### ISSUE TITLE
I cannot get it to work with flow

### ISSUE THREAD

**Issue by ghost** [storybooks/babel-plugin-react-docgen#43]
This is my package.json

```
    "babel-plugin-react-docgen": "^1.8.2",
    "@storybook/addon-actions": "^3.3.12",
    "@storybook/addon-backgrounds": "^3.3.12",
    "@storybook/addon-info": "^3.3.12",
    "@storybook/addon-knobs": "^3.3.12",
    "@storybook/addon-links": "^3.3.12",
    "@storybook/addons": "^3.3.12",
    "@storybook/react": "^3.3.12",
```

babelrc
```
{
  "presets": ["env", "react"],
  "plugins": ["react-docgen", "transform-class-properties"]
}

```

Component
```
// @flow
import React, { type Node } from 'react'
type Props = {
  children: Node,
}

const Component = ({ children, vertical, horizontal }: Props) => (
  <div>
    {children}
  </div>
)

Component.displayName = 'Component'

export default Component
```

but `Component.__docgenInfo` is always undefined. 

I am not sure what i am doing wrong.

**Comment by danielduan** [comment]
I wonder if flow interferes with the AST parser that we use to determine if it's a react component or not. Sounds like this is an edge case we missed.

**Comment by danielduan** [comment]
v1.9.0 published with docgen@3.0-beta. clear your lock files, reinstall, and let me know if this doesn't fix the issue.
